dist/proguard.txt

```diff
@@ -0,0 +1,4 @@
+-dontwarn **
+-dontnote **
+
+-keep @io.envoyproxy.envoymobile.Native class * { *; }
```

Comment:
This rule says to keep the class if it's annotated with `@Native`, but I think you'd want to keep only individual methods if they're annotated

Here is how Android's `@Keep` works:


```bash
# It's an example from Android SDK, actual Androidx annotation rules are in androidx.annotations aar file
cat "$ANDROID_HOME/tools/proguard/examples/annotations/lib/annotations.pro"
```

```console
# The following annotations can be specified with classes and with class
# members.

# @Keep specifies not to shrink, optimize, or obfuscate the annotated class
# or class member as an entry point.

-keep @proguard.annotation.Keep class *

-keepclassmembers class * {
    @proguard.annotation.Keep *;
}
```

Reply:
So `keep` doesn't work for some reason